scripts/experimental/test_battery.py

```diff
@@ -1,56 +1,75 @@
 #test battery.py
+# by M Vegh, last modified 1/26/2015
 import sys
 sys.path.append('../trunk')
 import SUAVE
 from SUAVE.Components.Energy.Storages.Battery import Battery
+from SUAVE.Attributes import Units
+from SUAVE.Methods.Power.Battery.Discharge import datta_discharge
+from SUAVE.Methods.Power.Battery.Sizing import initialize_from_energy_and_power, initialize_from_mass
+from SUAVE.Structure import Data
+from SUAVE.Methods.Power.Battery.Ragone import find_ragone_properties, find_specific_power, find_ragone_optimum
+from SUAVE.Methods.Power.Battery.Variable_Mass import find_mass_gain_rate, find_total_mass_gain
 import numpy as np
 def main():

```

Comment:
add some prettyfication header comments here, check out some of the other test cases